add-ons/grafana/README.md

```diff
@@ -0,0 +1,39 @@
+# Grafana Add-on
+An addon that will deploy grafana with hawkular and prometheus datasources.
+It requires to connect to hawkular-metrics, provided by management-infra addon.
+
+## Deploy grafana
+To deploy grafana do:
+
+```
+minishift addon apply grafana --addon-env namespace=grafana
+```
+
+_NOTE_: You should provide the namespace where it will be installed with the addon-env namespace.
+
+## Use grafana
+grafana will be available at:
+
+```
+$ minishift openshift service hawkular-grafana -n <namespace>
+```
+
```

Comment:
Once the grafana is opened in browser(after some time as it took time), I need to login with `user: admin, password: admin`. Probably, add it after above step.

Reply:
Added in new commit.